crates/sound/src/vhu_sound.rs

```diff
@@ -193,16 +204,21 @@ impl VhostUserSoundThread {
                     let start_id: usize = u32::from(query_info.start_id) as usize;
                     let count: usize = u32::from(query_info.count) as usize;
                     let mut pcm_info = vec![VirtioSoundPcmInfo::default(); count];
```

Comment:
Does it make sense to create a vec with Default values then changing all the values afterwards? I would instead do `Vec::with_capacity(count)` then append stuff when you iterate over stream_info.

Reply:
No, it does not. Thanks for the comment.